src/game_state.cpp

```diff
@@ -216,8 +219,6 @@ void game_state::init(const config& level, play_controller & pc)
 		}
 	}
 
-	pathfind_manager_.reset(new pathfind::manager(level));
-
 	lua_kernel_.reset(new game_lua_kernel(nullptr, *this, pc, *reports_));
```

Comment:
I'm a little confused, doesn't this line undo everything you were trying to do here?